app/assets/javascripts/component_guide/application.js

```diff
@@ -1,2 +1,11 @@
+/* global $ GOVUK */
+//= require jquery/dist/jquery
 //= require_tree ./vendor
 //= require_tree .
+//= require ../govuk_publishing_components/modules
+//= require ../govuk_publishing_components/components/accordion
+//= require ../govuk_publishing_components/components/tabs
+
+$(document).ready(function () {
+  GOVUK.modules.start()
+})
```

Comment:
```suggestion
//= require ../govuk_publishing_components/dependencies
//= require ../govuk_publishing_components/components/accordion
//= require ../govuk_publishing_components/components/tabs
```

Instead of importing jQuery and modules.js separately then initialising the modules script, we can follow our [generic guidance](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/install-and-use.md#import-javascript-for-individual-components) and call dependencies, which covers all these. Also not sure we _need_ to call `vendor` explicitly – that was meant to provide polyfills to be imported at the component level.

Reply:
I knew there must be a cleaner way of doing this, thanks, will update.